plugin/helpers.go

```diff
@@ -74,3 +83,28 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+func (p *HelpersImpl) InstallPluginFromUrl(downloadUrl string, replace bool) (*model.Manifest, error) {
```

Comment:
Would you please enforce the minimum server version like in https://github.com/mattermost/mattermost-server/blob/master/plugin/helpers_kv.go#L14-L17?

Reply:
Sure. Even before that can you please go through the conversation: https://community-release.mattermost.com/core/pl/fq1pk7qdfb8z3mqy515b1fnz6o. I am unsure what are we going forward ahead with.